src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewComponentResult.cs

```diff
@@ -47,17 +37,32 @@ public class ViewComponentResult : ActionResult
         public Type ViewComponentType { get; set; }
 
         /// <summary>
+        /// Get the view data model.
+        /// </summary>
+        public object Model => ViewData?.Model;
```

Comment:
`ViewComponentResult` probably has had the least use of all of these classes (`ViewResult`, `PartialViewResult`, etc). Is there anything else missing here or are all of these at rough parity?